nexus/social_media/services.py

```diff
@@ -0,0 +1,36 @@
+# Third Party Stuff
+import facebook
+from django.conf import settings
+
+# nexus Stuff
+from nexus.base import exceptions
+# Nexus Stuff
+from nexus.social_media.models import Post
+
+
+def get_fb_page_graph():
+    graph = facebook.GraphAPI(settings.FB_USER_ACCESS_TOKEN)
+    accounts = graph.get_object('me/accounts')['data']
+    page_access_token = None
+    for account in accounts:
+        if account['id'] == settings.FB_PAGE_ID:
+            page_access_token = account['access_token']
+            break
```

Comment:
Why is there a break here? what does the current check implies?

Reply:
The code currently posts on a page. So,
Use of break: To stop the loop when we get to the page with id same as that provided by the user.
Current check: Use to find page with id same as that provided by user. This is used to fetch page_access_token (required to post on page)
I should change the variable from account to page. That'll clear up the meaning a lot.

Reply:
As I can interpret from the code, `page` is a dictionary with key as the `page_id` and value being the `access_token`. Can you also name it something else? Does that depict something about what it actually contains? Like `pages_data` or something.

Why not simply check:

`list(filter(lambda page: page['name'] == settings.FB_PAGE_ID, page))` and if you get the result then fetch the `access_token`